Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj

```diff
@@ -56,12 +57,13 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>
     </NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This is necessary? 
I found only one `unsafe` block in XF rero in `GTK` gallery
https://github.com/xamarin/Xamarin.Forms/blob/bd31e1e9fc8b2f9ad94cc99e0c7ab058174821f3/Xamarin.Forms.ControlGallery.GTK/AdvancedOpenGLView.cs#L170

Reply:
I think so. That file is also used (file linking) in iOS and Android projects. OpenGL for the world, baby! 🌍